magefiles/magefile.go

```diff
@@ -593,7 +593,7 @@ func SetupMultiPlatformTests() error {
 				currentBuildahTaskRef = lastBundle.Value.StringVal
 				klog.Infof("Found current task ref %s", currentBuildahTaskRef)
 				//TODO: current use pinned sha?
-				lastBundle.Value = *tektonapi.NewStructuredValues("quay.io/redhat-appstudio-tekton-catalog/task-buildah-remote:0.1")
+				lastBundle.Value = *tektonapi.NewStructuredValues("quay.io/redhat-appstudio-tekton-catalog/task-buildah-remote:0.1-ac185e95bbd7a25c1c4acf86995cbaf30eebedc4")
```

Comment:
should we use fixed tag here? if we use fixed tag, we need to change it frequently to keep up the testing with latest `buildah-remote` task changes, which I guess often gets changed with `buildah` task. Rather I think we should run one basic `multi-platform` tests in the build-definitions repo CI to avoid breaking the MP tests.. to avoid similar issues in the future. WDYT?

Reply:
This was a hot fix to prevent CI from failing in this repo and infra-deployments.

I agree that we should rather cover the buildah-remote task with test, but not sure if it's a good idea to introduce some other test to already super slow and flakey build-definitions tests.

Could we instead update one of the test scenarios in build-definitions to use `buildah-remote` task instead of the regular `buildah`? Or do we have other options?

Reply:
we have created [a story ](https://issues.redhat.com/browse/STONEBLD-2303) now in the build team to address the issue

Reply:
> Could we instead update one of the test scenarios in build-definitions to use buildah-remote task instead of the regular buildah? 
I think it would be the better option.